Capri.Services/SystemSettings/SystemSettingsGetter.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.Extensions.Options;
+
+namespace Capri.Services.SystemSettings
+{
+    public class SystemSettingsGetter : ISystemSettingsGetter
+    {
+        private int _maxNumOfMasterProposalsPerPromoter;
+        private int _maxNumOfBachelorProposalsPerPromoter;
+
+        public SystemSettingsGetter(
+            IOptionsMonitor<SystemSettings> settingsMonitor)
+        {
+            settingsMonitor.OnChange(vals => {
```

Comment:
Quite a nice feature! Didn't know that something like that exists in ASP.NET Core. But one thing to mention:
Actually that should be set in web project, because it's strongly dependent on asp.net project. I think you can treat it as automapper configuration. You should move it to same dir as automapper config :)